Makefile

```diff
@@ -532,7 +532,7 @@ pytest_codeblocks_one:
 .PHONY: shell_lint # Lint all bash scripts
 shell_lint:
 	@# grep -v "^\./\." is to avoid files in .hidden_directories
-	find . -type f -name "*.sh" | grep -v "^\./\." | \
+	find . -type f -name "*.sh" | grep -v "^\./\." | grep -v "^\./playground/" | \
```

Comment:
I had the same issue, and changed my `playground` directory with `.playground` (which I then added in the gitignore), I think it's a better approach

Reply:
got it thanks!

Reply:
I would remove this added part to avoid future issues then !

Reply:
Yes done thanks

Reply:
I still see it 🤔